### PR TITLE
Html and the web div salad

### PR DESCRIPTION
**Pull request by Scot-MacDonald**

### CODE REVIEW

File: html-and-the-web/html-and-the-web_div-salad/index.html [hunk 1]
```diff
@@ -13,59 +13,59 @@
...
+          </figcaption>
+        </figure>
+      </section>
 
-      <div id="contact">
-        <div>Contact</div>
+      <section id="contact">
+        <h2>Contact</h2>
         <!--When clicking on these element, nothing happens! But I want to be redirected to the other websites. Why is it not working?-->
         <div href="https://github.com/spiced-academy">GitHub</div>
```
Comment (utkugunal) [inline html-and-the-web/html-and-the-web_div-salad/index.html]: I think, it is better to use <a href... instead of <div href... in these 3 lines.

**Review by utkugunal — COMMENTED** [review]
good job. I have only one comment, see below.

File: html-and-the-web/html-and-the-web_div-salad/index.html (hunk 1 above)
Comment (Scot-MacDonald) [reply html-and-the-web/html-and-the-web_div-salad/index.html]: arrrggghhh I thought I done that....Thanks Utku, I made the changes. Please approve when you have time.

Comment (utkugunal) [reply html-and-the-web/html-and-the-web_div-salad/index.html]: @Scot-MacDonald all seems fine to me now, you can merge it :)

**Pull request merged**